Core/include/Acts/Seeding/SeedFinderOrthogonalConfig.hpp

```diff
@@ -127,6 +123,16 @@ struct SeedFinderOrthogonalConfig {
 
     return config;
   }
+  SeedFinderOrthogonalConfig calculateDerivedQuantities() const {
+    SeedFinderOrthogonalConfig config = *this;
+    // calculation of scattering using the highland formula
+    // convert pT to p once theta angle is known
+    config.highland = 13.6 * std::sqrt(radLengthPerSeed) *
+                      (1 + 0.038 * std::log(radLengthPerSeed));
+    const float maxScatteringAngle = config.highland / minPt;
+    config.maxScatteringAngle2 = maxScatteringAngle * maxScatteringAngle;
+    return config;
+  };
```

Comment:
I need a clarification here @tboldagh and @stephenswat 
Are these variables supposed to be computer with config in internal units? 

As of now (i.e. before my changes) the orthogonal seeding is computing these BEFORE converting to internal units. That means that minPt is 900 (entered as MeV in JO - but 1 MeV = 0.1 in Acts units).

My code computes these quantities AFTER converting to internal units, and this minPt is 0.9 (Correctly MeV). This produces the differences in the physics performances as posted by the bot.

Do you have comments here? I'd suppose my approach is the correct one, but it comes with a change in performance (and much less seeds btw)

Reply:
Hi Carlo, when you say a change in performance do you mean a change in physics performance or a change in compute performance?

It seems to me your logic makes sense. This might also explain why this seeded sometimes produces many more seeds than the classic seeding algorithm.

Reply:
Concerning physics performance, the main effect is the duplicate rate: https://herald.dokku.paulgessinger.com/view/acts-project/acts/510272465/seeding_orthogonal.html 

The number of holes as well is effected: https://herald.dokku.paulgessinger.com/view/acts-project/acts/510272465/ckf_orthogonal.html 

See the physmon plots posted in the PR for all the details

Reply:
Concerning the number of produced seeds we go from 2616 to 1655 (63%)

Running on lxplus 
100 events
n threads: -1

Reply:
![timing](https://user-images.githubusercontent.com/75478407/212564330-68a8e13f-ec25-42c0-9efd-02a9d78ca1eb.png)
Quickly-performed study on timing. Not surprisingly this is faster

Reply:
That is an excellent result! We should go ahead with this!

Reply:
So we found it twice, see there, comment 2:
https://github.com/acts-project/acts/pull/1757
It could have spared some @CarloVarni  time if the other PR wold have been merged already.

Reply:
Hi @tboldagh I see. I suppose then your PR fully superseeds this one?

Reply:
> So we found it twice, see there, comment 2:
> #1757
> It could have spared some @CarloVarni time if the other PR wold have been merged already.

Trying my best ✌️

Reply:
Not sure. But maybe since mine is forgotten and this one got the wind we go ahead with this one? Either is fine.